File: 20260317_Contenido/20260317_Contenido_1_info.js
/*
Video: youtube.com/shorts/jsoQiiqzodQ

Cómo retornar varios valores con JavaScript:

Podemos usar una coma para separar los valores en corchetes que queremos retornar después de la palabra clave return. Y luego mandamos los datos en la tupla a la consola por medio de un ciclo for.
*/

function frase() {
  let cadena1 = "Hola";
  let cadena2 = "¿Cómo están?";

  return [cadena1, cadena2];
}

let matriz = frase();
for (let i of matriz) {
    console.log(i);
}

File: 20260317_Contenido/20260317_Contenido_1_info.py
"""
Video: youtube.com/shorts/49OuYPNXdwA

Cómo retornar varios valores con Python:

Podemos usar una coma para separar los valores que queremos retornar después de la palabra clave return. Y luego imprimimos los datos en la tupla por medio de un ciclo for.
"""

def frase():
    cadena1 = "Hola"
    cadena2 = "¿Cómo están?"

    return cadena1, cadena2

lista = frase()
for i in lista:
    print(i)
